<?xml version="1.0" encoding="utf-8"?>
<launch>

  <arg name="turtlebot_name"  default="unknown_node" />

  <node name="random_goals_publisher" pkg="turtlebot_control"
        type="random_goals_publisher">
    <param name="randMin" value="-1.0" />
    <param name="randMax" value="1.0"  />
    <param name="loopRate" value="1.0" />

    <remap from="start_flag_from_master" to="$(arg turtlebot_name)/start_flag_from_master"/>
  </node>

  <node name="npc_node_conveyor" pkg="turtlebot_control" type="npc_node_conveyor">
    <remap from="stop_flag_from_master" to="$(arg turtlebot_name)/stop_flag_from_master"/>
    <remap from="clear_flag_from_master" to="$(arg turtlebot_name)/clear_flag_from_master"/>
    <param name="goalFromUI" value="random_goal" />
    <param name="goalForTurtlebot" value="simple_goal" />
  </node>


</launch>
